RecoTauTag/TauTagTools/plugins/TGraphWriter.cc

```diff
@@ -0,0 +1,60 @@
+#include "RecoTauTag/TauTagTools/plugins/TGraphWriter.h"
```

Comment:
same about TGraph: please move to CondFormats/PhysicsToolsObjects/test or a similar appropriate place in the Cond* sybsystem